src/main/java/com/coscale/sdk/client/commons/Options.java

```diff
@@ -72,4 +72,17 @@ public String query() {
         parts.addAll(expandList());
         return Joiner.on('&').join(parts);
     }
+
+    /**
+     * If neither the selectBys or expands has entries, there are no options for the query.
+     * @return False if neither selectBys or expands has entries/is initialized.
+     * True if either one has entries.
+     */
+    public boolean hasQuery() {
```

Comment:
Check to see the possibilities. If no options are defined, return false. Return true if at least one of the options has items.